components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -402,22 +405,26 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
                         .toLowerCase().contains("redirect_uri=")) {
                     authzRequest = OAuthClientRequest.authorizationLocation(authorizationEP).setClientId(clientId)
                             .setResponseType(OIDCAuthenticatorConstants.OAUTH2_GRANT_TYPE_CODE).setState(state)
+                            .setParameter(NONCE, (String) context.getProperty(NONCE))
```

Comment:
lets extract this `(String) context.getProperty(NONCE)` to a variable before the if condition and then use that variable.

Reply:
Fixed by - https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/139/commits